CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+# Contributing to dGrants
+
+A big welcome and thank you for considering contributing to dGrants open source projects! It’s people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing these open source projects. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+## Quicklinks
+
+* [Code of Conduct](#code-of-conduct)
+* [Getting Started](#getting-started)
+    * [Discussions](#discussions)
+    * [Issues](#issues)
+    * [Pull Requests](#pull-requests)
+      * [Git Rebase Workflow](#git-rebase-workflow)
+* [Getting Help](#getting-help)
+
+## Code of Conduct
+
+Contributions to Gitcoin are governed by the [Contributor Covenant version 1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html).
+All contributors and participants agree to abide by its terms. To report
+violations, shoot out an email to founders@gitcoin.co
```

Comment:
"shoot out an email to founders@gitcoin.co" feels like it's not the reporting process we want for decentralized grants, but I don't have a good alternative to propose 🤔

Reply:
Yes, for the moment we can set it to my email. We could use a decentralized email account soon. I'm thinking https://ethmail.cc/ using the multisig potentially although I'm not sure how we would log in with the multisig.